lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +232,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        container_div = article_bs.find('div', class_='col-lg-9 col-md-12')
+        for link in container_div.find_all('a'):
+            if link.get('href').startswith('/news/202'):
+                href = 'https://www.gazetavechorka.ru' + link.get('href')
+                if href and href not in self.urls:
+                    self.urls.append(href)
```

Comment:
extract URL does not change the state of the instance. only does simple thing: takes/constructs URL based on page content.

Reply:
fixed